script/openqa-webui-daemon

```diff
@@ -1,3 +1,38 @@
-#!/bin/sh -e
-# Our API commands are very expensive, so the default timeouts are too tight
-exec "$(dirname "$0")"/openqa prefork -m production --proxy -i 100 -H 900 -w 30 -c 1 -G 800 "$@"
+#!/bin/bash
+set -e
+
+pid=
+pid_idx=0
+pid_dir=${OPENQA_BASEDIR:-/var/lib}/openqa/webui
+openqa_args=("$@")
+openqa_dir=$(dirname "$0")
```

Comment:
strictly speaking this is the script dir and not openqa dir ;) Of course both can be the same if the package installs it later into the openqa directory.

Reply:
I called it that way because it is the dir used for the openQA invocation in consistency with `openqa_args`.